packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -39,6 +39,17 @@ const releaseCardMediumListSx = {
   gap: '10px',
   mb: '30px',
 }
+const multiCardListSx = {
+  display: 'grid',
+  gridTemplateColumns: 'repeat(auto-fit, minmax(300px, 1fr))',
+  gridColumnGap: '20px',
+  marginBottom: '20px',
+  marginTop: '20px',
```

Comment:
Note - In the codebase you may see the shorthand `mb` used for `marginBottom`, `mt` for `marginTop`, and `m` for `margin`